pom.xml

```diff
@@ -112,16 +112,18 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <!--
+            FIX ME: Have to comment out this plugin since it conflicts with ForkJoinPool dependency.
```

Comment:
I'm very happy that you raised up this question. I spent some time while working on plugin framework and was afraid of inadmissibility this decision because of some reasons. If you don't mind of upping Java version to 7 I completely agree!!!